src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -126,28 +126,69 @@ cdef class SSLSessionCacheLRU:
     grpc_shutdown()
 
 
+cdef int verify_peer_callback_wrapper(const char* servername, const char* cert, void* userdata) with gil:
```

Comment:
How long is this line horizontally? I think you may need to line-wrap immediately after the opening parenthesis.

(Sorry that we don't have an autoformatter running on our Cython code!)